src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  width: 100%;
+}
+
+.search {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 70px;
```

Comment:
don`t hardcode height, try to use vertical paddings on input

Reply:
fix it